src/pages/developing/frameworks/vanilla.mdx

```diff
@@ -22,6 +22,9 @@ HTML and SCSS partials to build websites and user interfaces. Adopting the
 library enables developers to use consistent markup, styles, and behavior in
 prototype and production work.
 
+For v10 documents please visit our
+[v10 documentation](https://v10.carbondesignsystem.com/developing/frameworks/vanilla).
+
```

Comment:
Have we been using a consistent pattern for these notices? Wasn't sure if this was more appropriate in an inline notification or not.

Reply:
Oh yes good Idea I’m not sure if there’s consistent patterns but I do agree with being inline notification